src/components/Status.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
+import '../assets/styles/status.css';
 
-const Status = (prop) => {
-  const { book } = prop;
+const Status = () => {
+  // const { book } = prop;
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.